invenio_query_parser/contrib/spires/parser.py

```diff
@@ -139,6 +139,18 @@ class SpiresSimpleQuery(UnaryRule):
     grammar = attr('op', [SpiresKeywordQuery, SpiresValueQuery])
 
 
+class RangeValue(LeafRule):
```

Comment:
Can you actually already use `RangeValue` and `RangeOp` from the `invenio_query_parser.parser`?

https://github.com/inveniosoftware/invenio-query-parser/blob/9f2d3ca5613521000ac0eadf4e13e2cb4cda3f23/invenio_query_parser/parser.py#L149-L154